test/controllers/trust.interrupt.controller.spec.ts

```diff
@@ -63,12 +62,15 @@ describe('Trust Interrupt controller', () => {
       };
       (mapCommonTrustDataToPage as any as jest.Mock).mockReturnValue(mockTrustData);
 
+      // When no session increment by 1
+      const noSessionTrustId = '1';
+
       get(mockReq, mockRes, mockNext);
 
       expect(mockRes.redirect).not.toBeCalled();
 
       expect(mapCommonTrustDataToPage).toBeCalledTimes(1);
-      expect(mapCommonTrustDataToPage).toBeCalledWith(mockAppData, trustId);
+      expect(mapCommonTrustDataToPage).toBeCalledWith(mockAppData, noSessionTrustId);
```

Comment:
If you agree with us not using trustData in the controller we shouldn't need any `mapCommonTrustDataToPage ` checks